boost.sh

```diff
@@ -1515,8 +1513,8 @@ EOF
 parseArgs "$@"
 
 if [[ -z $BOOST_VERSION ]]; then
-    BOOST_VERSION=`curl -s 'https://www.boost.org' | sed -n 's/.*https:\/\/dl\.bintray\.com\/boostorg\/release\/\([^\/]*\)\/.*$/\1/p'`
-    echo "Detecting the latest boost version from https://www.boost.org to be version $BOOST_VERSION"
+    BOOST_VERSION=`curl -s 'https://github.com/boostorg/boost/releases' | grep -o "\/boostorg\/boost\/releases\/tag\/boost-\([0-9]\+\.[0-9]\+\.[0-9]\+\)\"" | cut -d"-" -f2 | cut -d"\"" -f1 | head -1`
```

Comment:
I'm querying the Github releases page for the latest version. At the moment the `boost.org` site is still referencing `dl.bintray.com` but they are currently working on moving this, but I didn't want to have to depend on them changing their public site information forms to scrape the latest version from. This will suffice.

Reply:
As I understand you plan to squash those commits?

Reply:
I will do a squash when this gets merged into `master`.